cloudflare/resource_cloudflare_ruleset_test.go

```diff
@@ -978,3 +1021,28 @@ func testAccCheckCloudflareRulesetManagedWAFWithIDBasedOverrides(rnd, name, zone
     }
   }`, rnd, name, zoneID, zoneName)
 }
+
+func testAccCheckCloudflareRulesetManagedWAFPayloadLoggigng(rnd, name, zoneID, zoneName string) string {
+	return fmt.Sprintf(`
+  resource "cloudflare_ruleset" "%[1]s" {
+    zone_id     = "%[3]s"
+    name        = "%[2]s"
+    description = "%[1]s ruleset description"
+    kind        = "zone"
+    phase       = "http_request_firewall_managed"
+
+    rules {
+      action = "execute"
+      action_parameters {
+        id = "efb7b8c949ac4650a09736fc376e9aee"
+        matched_data {
+					public_key = "not_a_real_public_key"
```

Comment:
Nit: the indentation looks odd in these two lines (using tabs, unlike the surrounding lines).